.github/workflows/lints.yml

```diff
@@ -4,33 +4,37 @@ on:
   push:
     branches:
       - 'main'
+      - 'develop'
   pull_request:
-    branches:
-      - 'main'
-      - '*/*'
   workflow_dispatch:
 
 jobs:
   lints:
     runs-on: ubuntu-latest
     timeout-minutes: 25
     steps:
-      - uses: styfle/cancel-workflow-action@0.11.0
-        name: Cancel Outdated Builds
+      - name: Cancel Outdated Builds
+        uses: styfle/cancel-workflow-action@0.11.0
         with:
           all_but_latest: true
           access_token: ${{ github.token }}
 
-      - name: Install Nix
-        uses: cachix/install-nix-action@v23
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y protobuf-compiler
+
+      - name: Install Just
+        run: |
+          wget https://github.com/casey/just/releases/download/1.14.0/just-1.14.0-x86_64-unknown-linux-musl.tar.gz
+          tar -vxf just-1.14.0-x86_64-unknown-linux-musl.tar.gz just
+          sudo cp just /usr/bin/just
```

Comment:
Nit: could we do `apt-get install -y just` here? If so maybe we could just bundle it in with the previous step

Reply:
( https://docs.makedeb.org/prebuilt-mpr/getting-started/ the apt repo info )

Reply:
Ah, makes sense!